editor/project_manager.cpp

```diff
@@ -55,7 +56,8 @@
 #include "servers/display_server.h"
 #include "servers/navigation_server_3d.h"
 
-static inline String get_project_key_from_path(const String &dir) {
+static inline String
+get_project_key_from_path(const String &dir) {
```

Comment:
Unexpected line break:
```suggestion
static inline String get_project_key_from_path(const String &dir) {
```